graphql_compiler/cost_estimation/statistics.py

```diff
@@ -159,7 +166,7 @@ def get_edge_count_between_vertex_pair(
             - int, the count of edges if the statistic exists
             - None otherwise
         """
-        statistic_key = (vertex_in_class_name, vertex_out_class_name, edge_class_name)
+        statistic_key = (vertex_out_class_name, vertex_in_class_name, edge_class_name)
```

Comment:
Please update the docs.

Reply:
The docs are correct, my code was just wrong :)

Reply:
Actually, do you mind changing the order? (Having `vertex_in_class_name` first). I find it more intuitive to write first the source and then the destination.

Reply:
I spoke with @obi1kenobi about this exact thing. I fully agree with you, and this is what vertex_out and vertex_in reflects. 
You can think of it this way: vertex_out is where the edge goes out of, and vertex_in is where the goes into.